app/views/ncr/work_orders/_attachments.html.haml

```diff
@@ -0,0 +1,6 @@
+.form-group
+  = field_set_tag 'Attachments' do
+    %ul.attachments{ data: { add_minus: true } }
+      - 10.times do
```

Comment:
Right now, `10` feels like a mystery number. I'm sure there is a reason behind allowing 10 attachments, but since I just see it here, it feels like maybe it is a typo or something random...what about using a constant to store the value?